tests/spec/jasmine-event-recur.js

```diff
@@ -207,6 +207,13 @@ describe("The Calendar Interval", function() {
         expect(recurrence.matches( moment(startDate).date(27) )).toBe(false);
     });
 
+    it("weeksOfMonthByDay should work", function() {
+        var recurrence = moment.recur().every(0).weeksOfMonthByDay();
+        expect(recurrence.matches(moment(startDate))).toBe(true);
+        expect(recurrence.matches(moment('2013-01-06'))).toBe(true);
+        expect(recurrence.matches(moment('2013-01-08'))).toBe(false);
+    });
```

Comment:
Yeah, I don't understand your expectaton here.  What does 'should work' mean in this case?  An english sentence can describe this much better.